event-processing/event-processor/tests/unit/app.test.ts

```diff
@@ -80,12 +74,7 @@ describe('Unit test for app handler', function () {
 
         await handler(sqsEvent);
 
-        expect(sns.publish).toHaveBeenCalledWith(
```

Comment:
Are we no longer able to test the publish request?

Reply:
No, because the event_id will now be enriched with a random uuid. Couldn't see any simple way to mock out just part of the json string.

Reply:
We could maybe mock the RandomUUID function to provide us with a known value.